tools/codegen/pkg/swaggerdocs/generator.go

```diff
@@ -74,12 +74,12 @@ func (g *generator) generateGroupVersion(groupName string, version generation.AP
 	if g.verify {
 		klog.V(2).Infof("Verifiying swagger docs for %s/%s", groupName, version.Name)
 
-		return verifySwaggerDocs(version.Name, outFilePath, docsForTypes)
+		return verifySwaggerDocs(version.PackageName, outFilePath, docsForTypes)
```

Comment:
that's interesting.

Reply:
There are some packages (looking at the image docker APIs) where the version is a non compliant string (`1.0`), which means the Go package name differs, everywhere else it is the same, `v1` and `v1`

I think technically the previous version of this was wrong, it was making an assumption that the go package matched the version, so this should be more stable going forward